subprojects/fftw3.wrap

```diff
@@ -0,0 +1,10 @@
+[wrap-file]
+directory = fftw-3.3.10
+source_url = http://fftw.org/fftw-3.3.10.tar.gz
+source_filename = fftw-3.3.10.tar.gz
+source_hash = 56c932549852cddcfafdab3820b0200c7742675be92179e59e6215b340e26467
+patch_directory = fftw3
+method = cmake
+

```

Comment:
Can't use cmake, the wrapdb is for native meson builds.

Reply:
Unfortunatelly, fftw uses magic to generate (yes, you heard it right) and build their code. I understand the "meson only" build policy but I will not be able to provide meson build system support in a reasonable amount of time. I will mark this PR as a draft and will update my contribution when I find time to tackle the bigger job...

Thank you!